api-gateway/wasmcloud.banking/client/apps/banking/src/features/dashboard/components/GraphCard.tsx

```diff
@@ -54,19 +54,19 @@ export function GraphCard({userInfo}: {userInfo: UserInformation}): React.ReactE
 }
 
 function RecentTransaction({transaction}: {transaction: Transaction}) {
-  const formattedDate = new Date(transaction.date * 1000).toLocaleString(undefined, {
-    dateStyle: 'medium',
-  });
+  // const formattedDate = new Date(transaction.date * 1000).toLocaleString(undefined, {
+  //   dateStyle: 'medium',
+  // });
```

Comment:
Do you need this?

Reply:
I kept it just so that if we add dates back to the table we can just uncomment